hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -202,6 +207,8 @@ public class AzureBlobFileSystemStore implements Closeable, ListingSupport {
 
   private final ExecutorService renameBlobExecutorService;
 
+  private final Integer ONE_MINUTE = 60;
```

Comment:
Nit: Can be moved to one of the constants file...

Reply:
This is not required. I have removed it.
+1 on the catch.